apps/storybook/src/TabbedCard.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  createGroupResponse,
+  createListGroupResponse,
+} from '@asap-hub/fixtures';
+import { GroupDataObject } from '@asap-hub/model';
+import { TabbedCard } from '@asap-hub/react-components';
+import { number } from '@storybook/addon-knobs';
+
+export default {
+  title: 'Molecules / Tabbed Card',
+};
+
+const groups = createListGroupResponse(number('Number of groups', 10)).items;
+
+const firstTab = {
+  tabTitle: `First Tab`,
+  items: groups,
+  createItem: (group: GroupDataObject) => (
+    <li key={`group-${group.id}`}>{group.name}</li>
```

Comment:
I am not sure this should be an LI. If its not a child of OR or UL its invalid HTML. If this is the case use a div